src/features/winterStoragePlaceDetails/queries.ts

```diff
@@ -0,0 +1,34 @@
+import gql from 'graphql-tag';
+
+export const WINTER_STORAGE_PLACE_DETAILS_QUERY = gql`
+  query WINTER_STORAGE_PLACE_DETAILS($placeId: ID!) {
+    winterStoragePlace(id: $placeId) {
+      id
+      isActive
+      number
+      width
+      length
+      leases {
+        edges {
+          node {
+            id
+            customer {
+              id
+              firstName
+              lastName
```

Comment:
Here, fetching customer's info separately using `CustomerName` might speed up the query this gets slow